grc/business_logic/data_structures/application_data.py

```diff
@@ -21,6 +22,7 @@ def any_duplicate_aws_file_names(uploads_files: List[EvidenceFile]):
 class ApplicationData:
     reference_number: str = None
     email_address: str = None
+    updated: datetime = None

```

Comment:
It's going to be confusing that this `application_data.updated` property is only set on one particular page (when the PDF bundle is created via the admin portal).
I think we should set it automatically whenever the ApplicationData object is loaded from the database.

I've raised a PR with a suggestion: https://github.com/cabinetoffice/grc-app/pull/169